src/openrct2/sprites.h

```diff
@@ -671,6 +671,9 @@ enum {
     SPR_RIDE_DESIGN_PREVIEW_MAZE_ICE_BLOCKS = 21992,
     SPR_RIDE_DESIGN_PREVIEW_MAZE_WOODEN_FENCES = 21993,
 
+    // Used by on-ride photo segment and station
+    SPR_METAL_PLATE = 22432,
```

Comment:
No, but we need to pick a name. Either `SPR_STATION_BASE_D` or `SPR_METAL_PLATE` or something more descriptive.